src/helpers/test/defaults.ts

```diff
@@ -0,0 +1,43 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import { RecordingStatus } from "../../common/types";
+import { IRecordingContext } from "../../contexts/RecordingContext";
+import { IStepsContext } from "../../contexts/StepsContext";
+import { IUrlContext } from "../../contexts/UrlContext";
+
+export const RECORDING_CONTEXT_DEFAULTS: IRecordingContext = {
```

Comment:
I wonder whether we should make these functions which return new objects every time to avoid mutation and the history of previous mocks interfering with other test's results.

Reply:
Sounds like a great idea. I'll work on this modification now.

Reply:
https://github.com/elastic/synthetics-recorder/pull/146/commits/3fb3e9399b0bffd8d48e923f04684a90a09ec273

LMK if anything looks wrong with this.